scss/components/_control-bar.scss

```diff
@@ -81,6 +81,13 @@
 
     .oo-control-bar-item:focus .oo-icon {
       @include highlight-base();
+      @include highlight-tab();
+      opacity: 1 !important;
+    }
+
+    .oo-control-bar-item:hover .oo-icon {
+      @include highlight-base();
+      @include remove-highlight-tab();
```

Comment:
I added the active state cause when you clicked on an icon the box-shadow appeared, then in the active state, I remove the box shadow.